pkg/resource/acl/custom_set_output.go

```diff
@@ -0,0 +1,93 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package acl
+
+import (
+	"context"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	svcapitypes "github.com/aws-controllers-k8s/memorydb-controller/apis/v1alpha1"
+)
+
+const (
+	// The number of minutes worth of events to retrieve.
+	// 14 days in minutes
+	eventsDuration = 20160
```

Comment:
Go has a `time` library that is great for time durations. I believe it's:
```
eventsDuration = time.Duration(14) * time.Days
```

Then later you can call
```
eventsDuration.Minutes()
```

Reply:
It doesn't have time.Days
Should I use
```
eventsDuration = time.Duration(336) * time.Hour
```
?

Reply:
I change to:
```
eventsDuration = time.Duration(336) * time.Hour
```
Then call it and convert to int64 to match the type.
```
int64(eventsDuration.Minutes())
```